src/main/resources/db/migration/V1_73__create_spotlight_submission_table.sql

```diff
@@ -0,0 +1,106 @@
+CREATE TABLE IF NOT EXISTS public.spotlight_submission (
+	id uuid NOT NULL,
+	grant_mandatory_questions_id uuid NULL,
+    grant_scheme int4 NULL,
+	status varchar(255) NOT NULL,
```

Comment:
did we agree that the enums would only be living in the backend?

Reply:
Spoke to Iain about this one, he wants them as strings rather than enums